lib/index.js

```diff
@@ -41,7 +41,8 @@ function processAttributes(t, attributes, root, templateElem) {
 			var attrName = attribute.name.name;
 
 			if (attribute.type === 'JSXAttribute') {
```

Comment:
Attributes without a value, (as in all boolean attributes) will not have a `value` but must still enter the below flow without throwing